### ISSUE TITLE
Validation vs. Test set for the burn scars dataset

### ISSUE THREAD

**Issue by gabrieltseng** [NASA-IMPACT/hls-foundation-os#40]
Hello!

Thank you for releasing these datasets.

I understand that you use the "validation set" to report test results in the paper. It also seems like you use this set as a "true" validation set:

https://github.com/NASA-IMPACT/hls-foundation-os/blob/df7761efdcedd187372b1c19bde113e2040bfd5b/configs/burn_scars.py#L141

Did you use the "val/test" set for both validation / hyperparameter selection and reporting test results?

If not, what split did you use for validation & hyperparameter selection?

Thanks!
